docs/design/versioning.md

```diff
@@ -84,9 +99,9 @@ Here is an example major release cycle:
 
 It may seem a bit strange to complete the v2 API before v2.0 is released, but *adding* a v2 API is not a breaking change. *Removing* the v2beta\* APIs *is* a breaking change, which is what necessitates the major version bump. There are other ways to do this, but having the major release be the fresh start of that release's API without the baggage of its beta versions seems most intuitive out of the available options.
 
-## Patch releases
+## Branched patch releases
```

Comment:
All patch releases are intended for critical bug fixes, etc.  Let's talk more about this.